styles/alpha.json

```diff
@@ -134,27 +134,27 @@
 			"fontSizes": [
 				{
 					"slug": "extra-small",
-					"size": "1rem",
+					"size": "clamp(1rem, 1rem, 1rem)",
 					"name": "Extra small"
 				},
 				{
 					"slug": "small",
-					"size": "1.125rem",
+					"size": "clamp(1.125rem, 1.125rem, 1.125rem)",
 					"name": "Small"
 				},
 				{
 					"slug": "normal",
-					"size": "1.25rem",
+					"size": "clamp(1.25rem, 1.25rem, 1.25rem)",
```

Comment:
I think we don't need the clamp here since it's all the same value. I feel like it should be something like this:

extra-small: `"size": "0.75rem" (this one has no equivalent bcds style from what I can tell)
small: `"size": "var(--bcds-typography-font-size-label)"`
normal: `"size": "var(--bcds-typography-font-size-body)"`
large:  `"size": "var(--bcds-typography-font-size-large-body)"'
extra-large: `"size": "var(--bcds-typography-font-size-h5)"'

This isn't ideal but I'm trying to use as much of the BCDS styles as we can while still being compatible with how the alpha theme is already being used on our sites.